2018-01-26 14:51:36,990 p=12178 u=niten2 |  PLAY ***************************************************************************
2018-01-26 14:51:36,998 p=12178 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 14:51:41,752 p=12178 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "  File \"/root/.ansible/tmp/ansible-tmp-1516953097.02-56318973060401/setup\", line 3160\r\n    except OSError, e:\r\n                  ^\r\nSyntaxError: invalid syntax\r\n", "msg": "MODULE FAILURE", "parsed": false}[0m
2018-01-26 14:51:41,753 p=12178 u=niten2 |  NO MORE HOSTS LEFT *************************************************************
2018-01-26 14:51:41,754 p=12178 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 14:51:41,754 p=12178 u=niten2 |  194.87.99.112              : ok=0    changed=0    unreachable=0    failed=1   
2018-01-26 14:54:08,641 p=13821 u=niten2 |  PLAY ***************************************************************************
2018-01-26 14:54:08,650 p=13821 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 14:54:12,155 p=13821 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "  File \"/root/.ansible/tmp/ansible-tmp-1516953248.68-63573544934010/setup\", line 3160\r\n    except OSError, e:\r\n                  ^\r\nSyntaxError: invalid syntax\r\n", "msg": "MODULE FAILURE", "parsed": false}[0m
2018-01-26 14:54:12,156 p=13821 u=niten2 |  NO MORE HOSTS LEFT *************************************************************
2018-01-26 14:54:12,156 p=13821 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 14:54:12,157 p=13821 u=niten2 |  194.87.99.112              : ok=0    changed=0    unreachable=0    failed=1   
2018-01-26 14:54:17,109 p=13910 u=niten2 |  PLAY ***************************************************************************
2018-01-26 14:54:17,117 p=13910 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 14:54:20,990 p=13910 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 14:54:20,994 p=13910 u=niten2 |  TASK [apt upgrade] *************************************************************
2018-01-26 15:07:49,981 p=13910 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:07:49,983 p=13910 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=0   
2018-01-26 15:08:11,608 p=14741 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:08:11,620 p=14741 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:08:22,987 p=14741 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:08:22,995 p=14741 u=niten2 |  TASK [apt upgrade] *************************************************************
2018-01-26 15:09:15,655 p=14741 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "msg": "'/usr/bin/apt-get dist-upgrade' failed: E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)\nE: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?\n", "stdout": "", "stdout_lines": []}[0m
2018-01-26 15:09:15,657 p=14741 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:09:15,657 p=14741 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 15:10:33,580 p=16324 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:10:33,590 p=16324 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:10:46,252 p=16324 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:10:46,255 p=16324 u=niten2 |  TASK [apt upgrade] *************************************************************
2018-01-26 15:10:56,795 p=16324 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "msg": "'/usr/bin/apt-get dist-upgrade' failed: E: Could not get lock /var/lib/dpkg/lock - open (11: Resource temporarily unavailable)\nE: Unable to lock the administration directory (/var/lib/dpkg/), is another process using it?\n", "stdout": "", "stdout_lines": []}[0m
2018-01-26 15:10:56,798 p=16324 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:10:56,798 p=16324 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 15:19:31,253 p=17269 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:19:31,266 p=17269 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:19:41,342 p=17269 u=niten2 |  [0;31mfatal: [194.87.99.112]: UNREACHABLE! => {"changed": false, "msg": "ERROR! SSH encountered an unknown error during the connection. We recommend you re-run the command using -vvvv, which will enable SSH debugging output to help diagnose the issue", "unreachable": true}[0m
2018-01-26 15:19:41,343 p=17269 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:19:41,344 p=17269 u=niten2 |  194.87.99.112              : ok=0    changed=0    unreachable=1    failed=0   
2018-01-26 15:31:59,296 p=18563 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:31:59,305 p=18563 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:32:01,761 p=18563 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "module_stderr": "", "module_stdout": "bash: /usr/bin/python: No such file or directory\r\n", "msg": "MODULE FAILURE", "parsed": false}[0m
2018-01-26 15:32:01,762 p=18563 u=niten2 |  NO MORE HOSTS LEFT *************************************************************
2018-01-26 15:32:01,762 p=18563 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:32:01,762 p=18563 u=niten2 |  194.87.99.112              : ok=0    changed=0    unreachable=0    failed=1   
2018-01-26 15:36:18,826 p=18839 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:36:18,835 p=18839 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:36:24,743 p=18839 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:36:24,752 p=18839 u=niten2 |  TASK [pip : Ensure Pip is installed.] ******************************************
2018-01-26 15:43:47,691 p=18839 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:43:47,692 p=18839 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=0   
2018-01-26 15:43:51,167 p=20167 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:43:51,176 p=20167 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:43:56,422 p=20167 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:43:56,425 p=20167 u=niten2 |  TASK [pip : Ensure Pip is installed.] ******************************************
2018-01-26 15:43:59,596 p=20167 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:43:59,598 p=20167 u=niten2 |  TASK [pip : Ensure pip_install_packages are installed.] ************************
2018-01-26 15:44:20,121 p=20167 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:44:20,123 p=20167 u=niten2 |  194.87.99.112              : ok=2    changed=0    unreachable=0    failed=0   
2018-01-26 15:44:21,199 p=20404 u=niten2 |  [0;31mERROR! 'var' is not a valid attribute for a Task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/pip/tasks/main.yml': line 8, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- debug:
  ^ here
[0m
2018-01-26 15:44:29,642 p=20474 u=niten2 |  [0;31mERROR! 'var' is not a valid attribute for a Task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/pip/tasks/main.yml': line 8, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- debug:
  ^ here
[0m
2018-01-26 15:44:49,796 p=20551 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/pip/tasks/main.yml': line 8, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - debug:
  ^ here
[0m
2018-01-26 15:44:58,153 p=20625 u=niten2 |  [0;31mERROR! 'var' is not a valid attribute for a Task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/pip/tasks/main.yml': line 8, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- debug:
  ^ here
[0m
2018-01-26 15:45:02,702 p=20694 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:45:02,711 p=20694 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:45:18,758 p=20694 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:45:18,766 p=20694 u=niten2 |  TASK [pip : Ensure Pip is installed.] ******************************************
2018-01-26 15:45:22,416 p=20694 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "msg": "unsupported parameter for module: become"}[0m
2018-01-26 15:45:22,418 p=20694 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:45:22,419 p=20694 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 15:45:38,690 p=20803 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:45:38,699 p=20803 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:45:58,529 p=20803 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:45:58,536 p=20803 u=niten2 |  TASK [docker : Install Docker.] ************************************************
2018-01-26 15:46:01,676 p=20803 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "msg": "No package matching 'docker-ce' is available"}[0m
2018-01-26 15:46:01,677 p=20803 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:46:01,677 p=20803 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 15:46:24,202 p=21161 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:46:24,214 p=21161 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:46:26,674 p=21161 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:46:26,677 p=21161 u=niten2 |  TASK [docker : include] ********************************************************
2018-01-26 15:46:26,719 p=21161 u=niten2 |  [0;36mskipping: [194.87.99.112][0m
2018-01-26 15:46:26,721 p=21161 u=niten2 |  TASK [docker : include] ********************************************************
2018-01-26 15:46:26,775 p=21161 u=niten2 |  [0;36mincluded: /home/niten2/code/go/src/app-telegram/ansible/roles/docker/tasks/setup-Debian.yml for 194.87.99.112[0m
2018-01-26 15:46:26,792 p=21161 u=niten2 |  TASK [docker : Ensure old versions of Docker are not installed.] ***************
2018-01-26 15:46:29,792 p=21161 u=niten2 |  [0;32mok: [194.87.99.112] => (item=[u'docker', u'docker-engine'])[0m
2018-01-26 15:46:29,794 p=21161 u=niten2 |  TASK [docker : Ensure depdencies are installed.] *******************************
2018-01-26 15:46:32,735 p=21161 u=niten2 |  [0;32mok: [194.87.99.112] => (item=[u'apt-transport-https', u'ca-certificates'])[0m
2018-01-26 15:46:32,743 p=21161 u=niten2 |  TASK [docker : Add Docker apt key.] ********************************************
2018-01-26 15:46:36,421 p=21161 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:46:36,425 p=21161 u=niten2 |  TASK [docker : Ensure curl is present (on older systems without SNI).] *********
2018-01-26 15:46:36,476 p=21161 u=niten2 |  [0;36mskipping: [194.87.99.112][0m
2018-01-26 15:46:36,478 p=21161 u=niten2 |  TASK [docker : Add Docker apt key (alternative for older systems without SNI).] 
2018-01-26 15:46:36,522 p=21161 u=niten2 |  [0;36mskipping: [194.87.99.112][0m
2018-01-26 15:46:36,525 p=21161 u=niten2 |  TASK [docker : Add Docker repository.] *****************************************
2018-01-26 15:48:21,393 p=21161 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:48:21,397 p=21161 u=niten2 |  TASK [docker : Install Docker.] ************************************************
2018-01-26 15:50:13,013 p=21161 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:50:13,016 p=21161 u=niten2 |  TASK [docker : Ensure Docker is started and enabled at boot.] ******************
2018-01-26 15:50:15,800 p=21161 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:50:15,804 p=21161 u=niten2 |  TASK [docker : include] ********************************************************
2018-01-26 15:50:15,838 p=21161 u=niten2 |  [0;36mincluded: /home/niten2/code/go/src/app-telegram/ansible/roles/docker/tasks/docker-compose.yml for 194.87.99.112[0m
2018-01-26 15:50:15,868 p=21161 u=niten2 |  TASK [docker : Check current docker-compose version.] **************************
2018-01-26 15:50:18,701 p=21161 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:50:18,703 p=21161 u=niten2 |  TASK [docker : Delete existing docker-compose version if it's different.] ******
2018-01-26 15:50:18,747 p=21161 u=niten2 |  [0;36mskipping: [194.87.99.112][0m
2018-01-26 15:50:18,749 p=21161 u=niten2 |  TASK [docker : Install Docker Compose (if configured).] ************************
2018-01-26 15:50:29,949 p=21161 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:50:29,965 p=21161 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:50:29,966 p=21161 u=niten2 |  194.87.99.112              : ok=11   changed=4    unreachable=0    failed=0   
2018-01-26 15:52:39,723 p=23081 u=niten2 |  [0;31mERROR! 'var' is not a valid attribute for a Task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 7, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- debug:
  ^ here
[0m
2018-01-26 15:53:41,025 p=23274 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:53:41,036 p=23274 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:53:48,117 p=23274 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:53:48,121 p=23274 u=niten2 |  TASK [app-telegram : create dir ~/app-telegram] ********************************
2018-01-26 15:53:49,759 p=23274 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:53:49,766 p=23274 u=niten2 |  TASK [app-telegram : debug] ****************************************************
2018-01-26 15:53:49,798 p=23274 u=niten2 |  [0;32mok: [194.87.99.112] => {
    "output": "VARIABLE IS NOT DEFINED!"
}[0m
2018-01-26 15:53:49,800 p=23274 u=niten2 |  TASK [app-telegram : create dir ~/app-telegram/etc] ****************************
2018-01-26 15:53:51,281 p=23274 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:53:51,286 p=23274 u=niten2 |  TASK [app-telegram : copy .env.production] *************************************
2018-01-26 15:53:53,964 p=23274 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:53:53,968 p=23274 u=niten2 |  TASK [app-telegram : copy go build app-telegram] *******************************
2018-01-26 15:53:59,614 p=23274 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:53:59,616 p=23274 u=niten2 |  TASK [app-telegram : copy docker-compose.yml] **********************************
2018-01-26 15:54:02,542 p=23274 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 15:54:02,545 p=23274 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:54:02,545 p=23274 u=niten2 |  194.87.99.112              : ok=7    changed=5    unreachable=0    failed=0   
2018-01-26 15:54:33,536 p=23455 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 36, column 13, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  debug:
      var: {{ output }}
            ^ here
We could be wrong, but this one looks like it might be an issue with
missing quotes.  Always quote template expression brackets when they
start a value. For instance:

    with_items:
      - {{ foo }}

Should be written as:

    with_items:
      - "{{ foo }}"
[0m
2018-01-26 15:54:40,027 p=23524 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 28, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: build docker
  ^ here
[0m
2018-01-26 15:54:50,099 p=23601 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 28, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: build docker
  ^ here
[0m
2018-01-26 15:55:13,456 p=23687 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 28, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: build docker
  ^ here
[0m
2018-01-26 15:55:28,068 p=23761 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 28, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: build docker images
  ^ here
[0m
2018-01-26 15:55:34,449 p=23829 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: build docker images
  ^ here
[0m
2018-01-26 15:55:39,613 p=23897 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 27, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: build docker images
  ^ here
[0m
2018-01-26 15:56:12,735 p=23993 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- docker_service:
  ^ here
[0m
2018-01-26 15:56:37,258 p=24079 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 33, column 1, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  build: yes
register: output
^ here
[0m
2018-01-26 15:56:41,764 p=24147 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 30, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- docker_service:
  ^ here
[0m
2018-01-26 15:56:46,986 p=24218 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 30, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - docker_service:
    ^ here
[0m
2018-01-26 15:57:13,849 p=24303 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 33, column 1, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  build: yes
register: output
^ here
[0m
2018-01-26 15:57:18,726 p=24377 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 30, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- docker_service:
  ^ here
[0m
2018-01-26 15:57:21,379 p=24446 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 30, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- docker_service:
  ^ here
[0m
2018-01-26 15:57:25,713 p=24514 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 30, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- docker_service:
  ^ here
[0m
2018-01-26 15:58:07,230 p=24618 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 30, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

- name: Run using a project directory
  - docker_service:
  ^ here
[0m
2018-01-26 15:58:11,690 p=24687 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 15:58:23,850 p=24767 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:58:23,859 p=24767 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:58:28,644 p=24767 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:58:29,630 p=24909 u=niten2 |  PLAY ***************************************************************************
2018-01-26 15:58:29,638 p=24909 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 15:58:32,954 p=24909 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 15:58:32,962 p=24909 u=niten2 |  TASK [app-telegram : Run using a project directory] ****************************
2018-01-26 15:58:32,991 p=24909 u=niten2 |  [0;32mok: [194.87.99.112] => {
    "output": "VARIABLE IS NOT DEFINED!"
}[0m
2018-01-26 15:58:32,993 p=24909 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 15:58:32,994 p=24909 u=niten2 |  194.87.99.112              : ok=2    changed=0    unreachable=0    failed=0   
2018-01-26 15:58:59,330 p=25009 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 15:59:08,853 p=25084 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 15:59:20,748 p=25159 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 33, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


  - docker_service:
  ^ here
[0m
2018-01-26 16:00:07,809 p=25322 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:00:10,635 p=25392 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 29, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:00:17,296 p=25468 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 3, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:00:19,754 p=25537 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:00:25,274 p=25606 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 3, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:00:34,870 p=25677 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 3, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:00:43,677 p=25750 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 8, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: Run using a project directory
      ^ here
[0m
2018-01-26 16:00:59,995 p=25898 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 9, column 23, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - name: Run using a project directory
        docker_service:
                      ^ here
[0m
2018-01-26 16:01:09,257 p=25968 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - name: Run using a project directory
      - docker_service:
      ^ here
[0m
2018-01-26 16:01:13,529 p=26095 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - name: Run using a project directory
      - docker_service:
      ^ here
[0m
2018-01-26 16:01:17,319 p=26170 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 8, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: Run using a project directory
      ^ here
[0m
2018-01-26 16:01:25,416 p=26243 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 9, column 21, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    name: Run using a project directory
      docker_service:
                    ^ here
[0m
2018-01-26 16:01:29,423 p=26311 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 8, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - docker_service:
      ^ here
[0m
2018-01-26 16:01:36,196 p=26383 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 8, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - docker_service:
      ^ here
[0m
2018-01-26 16:04:12,490 p=26657 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 7, column 8, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
     - docker_service:
       ^ here
[0m
2018-01-26 16:04:58,210 p=26757 u=niten2 |  [0;31mERROR! Syntax Error while loading YAML.


The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 33, column 2, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

- name: Run using a project directory
 tasks:
 ^ here
[0m
2018-01-26 16:05:10,453 p=26826 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 32, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:05:27,480 p=26907 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 31, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:05:32,755 p=26976 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/roles/app-telegram/tasks/main.yml': line 31, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Run using a project directory
  ^ here
[0m
2018-01-26 16:05:55,455 p=27066 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 8, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: Run using a project directory
      ^ here
[0m
2018-01-26 16:06:18,012 p=27141 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 7, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
    - docker_service:
      ^ here
[0m
2018-01-26 16:06:38,616 p=27226 u=niten2 |  [0;31mERROR! no action detected in task

The error appears to have been in '/home/niten2/code/go/src/app-telegram/ansible/setup.yml': line 9, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  tasks:
    - docker_service:
      ^ here
[0m
2018-01-26 16:09:41,579 p=27616 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:09:41,603 p=27616 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:09:45,831 p=27616 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:09:45,837 p=27616 u=niten2 |  TASK [app-telegram : docker-compose build] *************************************
2018-01-26 16:09:50,293 p=27616 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": true, "cmd": "docker-compose build", "delta": "0:00:02.816929", "end": "2018-01-26 12:09:50.191838", "failed": true, "rc": 1, "start": "2018-01-26 12:09:47.374909", "stderr": "mongo uses an image, skipping\nBuilding app\nCannot locate specified Dockerfile: Dockerfile", "stdout": "", "stdout_lines": [], "warnings": []}[0m
2018-01-26 16:09:50,297 p=27616 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:09:50,297 p=27616 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 16:10:21,243 p=27734 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:10:21,258 p=27734 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:10:24,365 p=27734 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:10:24,377 p=27734 u=niten2 |  TASK [app-telegram : copy docker-compose.yml] **********************************
2018-01-26 16:10:27,561 p=27734 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 16:10:27,565 p=27734 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:10:27,565 p=27734 u=niten2 |  194.87.99.112              : ok=2    changed=1    unreachable=0    failed=0   
2018-01-26 16:10:32,439 p=27826 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:10:32,448 p=27826 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:10:35,309 p=27826 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:10:35,318 p=27826 u=niten2 |  TASK [app-telegram : docker-compose build] *************************************
2018-01-26 16:20:32,905 p=27826 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:20:32,907 p=27826 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=0   
2018-01-26 16:33:44,003 p=30341 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:33:44,012 p=30341 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:33:53,221 p=30341 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:33:53,233 p=30341 u=niten2 |  TASK [app-telegram : copy go build app-telegram] *******************************
2018-01-26 16:34:48,296 p=30341 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"failed": true, "msg": "ERROR! error while accessing the file /home/niten2/code/go/src/app-telegram/etc/mongo/WiredTiger.turtle, error was: [Errno 13] Permission denied: u'/home/niten2/code/go/src/app-telegram/etc/mongo/WiredTiger.turtle'"}[0m
2018-01-26 16:34:48,297 p=30341 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:34:48,298 p=30341 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 16:35:12,466 p=30604 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:35:12,474 p=30604 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:35:15,173 p=30604 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:35:15,183 p=30604 u=niten2 |  TASK [app-telegram : copy go build app-telegram] *******************************
2018-01-26 16:35:25,202 p=30604 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 16:35:25,204 p=30604 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:35:25,205 p=30604 u=niten2 |  194.87.99.112              : ok=2    changed=1    unreachable=0    failed=0   
2018-01-26 16:36:22,133 p=30859 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:36:22,147 p=30859 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:36:25,261 p=30859 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:36:25,267 p=30859 u=niten2 |  TASK [app-telegram : copy go build app-telegram] *******************************
2018-01-26 16:36:34,406 p=30859 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 16:36:34,407 p=30859 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:36:34,408 p=30859 u=niten2 |  194.87.99.112              : ok=2    changed=1    unreachable=0    failed=0   
2018-01-26 16:53:24,660 p=1409 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:53:24,669 p=1409 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:53:27,016 p=1409 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:53:38,185 p=1555 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:53:38,194 p=1555 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:53:49,795 p=1555 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:53:49,801 p=1555 u=niten2 |  TASK [app-telegram : copy docker-compose.yml] **********************************
2018-01-26 16:53:52,732 p=1555 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 16:53:52,737 p=1555 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:53:52,738 p=1555 u=niten2 |  194.87.99.112              : ok=2    changed=1    unreachable=0    failed=0   
2018-01-26 16:55:24,484 p=1786 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:55:24,493 p=1786 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:55:28,497 p=1786 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:55:28,504 p=1786 u=niten2 |  TASK [app-telegram : copy .env.production] *************************************
2018-01-26 16:55:31,312 p=1786 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 16:55:31,315 p=1786 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:55:31,315 p=1786 u=niten2 |  194.87.99.112              : ok=2    changed=1    unreachable=0    failed=0   
2018-01-26 16:57:04,392 p=2063 u=niten2 |  PLAY ***************************************************************************
2018-01-26 16:57:04,401 p=2063 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 16:57:08,384 p=2063 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 16:57:08,390 p=2063 u=niten2 |  TASK [app-telegram : add mongodb user] *****************************************
2018-01-26 16:57:09,866 p=2063 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "msg": "the python pymongo module is required"}[0m
2018-01-26 16:57:09,868 p=2063 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 16:57:09,868 p=2063 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 17:00:00,253 p=2395 u=niten2 |  PLAY ***************************************************************************
2018-01-26 17:00:00,261 p=2395 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 17:00:04,683 p=2395 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 17:00:04,689 p=2395 u=niten2 |  TASK [app-telegram : add mongodb user] *****************************************
2018-01-26 17:00:06,896 p=2395 u=niten2 |  [0;31mfatal: [194.87.99.112]: FAILED! => {"changed": false, "failed": true, "msg": "The localhost login exception only allows the first admin account to be created"}[0m
2018-01-26 17:00:06,900 p=2395 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 17:00:06,900 p=2395 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=1   
2018-01-26 17:04:04,638 p=2790 u=niten2 |  PLAY ***************************************************************************
2018-01-26 17:04:04,647 p=2790 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 17:04:11,807 p=2790 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 17:04:11,811 p=2790 u=niten2 |  TASK [app-telegram : copy .env.production] *************************************
2018-01-26 17:04:21,937 p=2790 u=niten2 |  [0;33mchanged: [194.87.99.112][0m
2018-01-26 17:04:21,943 p=2790 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 17:04:21,943 p=2790 u=niten2 |  194.87.99.112              : ok=2    changed=1    unreachable=0    failed=0   
2018-01-26 17:05:31,312 p=2958 u=niten2 |  PLAY ***************************************************************************
2018-01-26 17:05:31,350 p=2958 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 17:05:35,406 p=2958 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 17:05:35,410 p=2958 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 17:05:35,411 p=2958 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=0   
2018-01-26 17:05:44,754 p=3045 u=niten2 |  PLAY ***************************************************************************
2018-01-26 17:05:44,779 p=3045 u=niten2 |  TASK [setup] *******************************************************************
2018-01-26 17:05:47,740 p=3045 u=niten2 |  [0;32mok: [194.87.99.112][0m
2018-01-26 17:05:47,744 p=3045 u=niten2 |  PLAY RECAP *********************************************************************
2018-01-26 17:05:47,745 p=3045 u=niten2 |  194.87.99.112              : ok=1    changed=0    unreachable=0    failed=0   
